src/controllers/controllerengine.h

```diff
@@ -198,6 +197,7 @@ class ControllerEngine : public QObject {
     double getDeckRate(const QString& group);
 
     Controller* m_pController;
+    UserSettingsPointer m_pConfig;
```

Comment:
It makes sense to declare all immutable members as `const`. I prefer to declare all const members before any mutable members if the initialization order allows this.